.gitattributes

```diff
@@ -1,3 +1,10 @@
 # Windows files should use crlf line endings
 # https://help.github.com/articles/dealing-with-line-endings/
 *.bat text eol=crlf
+
+*.snap linguist-generated=true
```

Comment:
Linguist files weren't set up on the original project, so I added the necessary configuration to simplify the modified files visualization.